frontend/public/locales/en/translation.json

```
@@ -1724,6 +1724,7 @@
   "machinePool.modal.scale.enable-autoscale.title": "Enable autoscale",
   "machinePool.modal.scale.maxReplicas.label": "Maximum replicas",
   "machinePool.modal.scale.minReplicas.label": "Minimum replicas",
+  "machinePool.modal.scale.minReplicas.tooltip": "When you enable autoscaling for the first time, the suggested value for minimum replicas is equal to the number of availability zones configured for the cluster. By default, these are all availability zones in the selected region, but a subset of zones can be selected during cluster creation for AWS and Azure.",
```

Apologies, I missed one more thing. 
```suggestion
  "machinePool.modal.scale.minReplicas.tooltip": "When you enable autoscaling for the first time, the suggested value for minimum replicas is equal to the number of availability zones configured for the cluster. By default, these are all availability zones in the selected region, but you can select a subset of zones during cluster creation for AWS and Azure.",
```